airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1919,6 +1936,13 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Notification"
+    WorkspaceGiveFeedback:
```

Comment:
Seems like this should include a boolean so it sets the feedback status instead of only marking it as done. 

Alternatively, if it only marks it as submitted, the name of the endpoint should reflect that.

Reply:
I change the endpoint name as `tag_feedback_status_as_done`. The lack of value is equivalent to having it as not done. We only set it to true and we don't put it back to false.